reactor-core/src/test/java/reactor/core/publisher/FluxUsingTest.java

```diff
@@ -146,6 +146,24 @@ public void normalEager() {
 		assertThat(cleanup).hasValue(1);
 	}
 
+	@Test
```

Comment:
I do understand we have tests for the regular cleanup, but it needs to be tested in case the implementations diverge at any point -> validating both eager and lazy cleanup behaviour in the scenario that the `AutoCloseable` throws.